prov/verbs/src/fi_verbs.h

```diff
@@ -580,11 +574,22 @@ struct vrb_ep {
 		};
 	};
 
-	size_t				inject_limit;
-
+	struct {
+		size_t			inject_size;
+		size_t                  tx_size;
+		size_t                  tx_iov_limit;
+		size_t                  rx_size;
+		size_t                  rx_iov_limit;
+		uint32_t                protocol;
+		uint32_t                addr_format;
+		size_t                  src_addrlen;
+		size_t                  dest_addrlen;
```

Comment:
I don't think there's ever a case where src/dest addrlen differ.  You may be able to use a single value here.

Reply:
Could this be considered as a future optimization?

Reply:
Yes.  I think as-is this is an improvement over what's there.